test/e2e/conformance_test.go

```diff
@@ -18,7 +18,7 @@ import (
 
 const (
 	relayerImageEnv    = "RELAYER_IMAGE"
-	relayerImage       = "strangeloveventures/relayer"
+	relayerImage       = "ghcr.io/strangelove-ventures/relayer"
```

Comment:
Perhaps unrelated for this PR but why are we using the strangelove relayer? guess it is the only one that has live docker images but will never work for this test

Reply:
Oh yeah, I left that unattendaed. I am overrding using env. I think its better to have somewhere like dockerhub.

I have working image pushed here. debendraoli/relayer.

Reply:
Yeah we probably should build one whenever ibc-relay gets a push to main in the future, but your own might be better as a default for now if you have it in a public dockerhub